src/data_types/cred_request.rs

```diff
@@ -11,8 +11,10 @@ pub struct CredentialRequest {
     #[serde(skip_serializing_if = "Option::is_none")]
     prover_did: Option<String>,
     cred_def_id: CredentialDefinitionId,
-    pub blinded_ms: ursa::cl::BlindedCredentialSecrets,
-    pub blinded_ms_correctness_proof: ursa::cl::BlindedCredentialSecretsCorrectnessProof,
+    #[serde(alias = "blinded_ms")]
+    pub blinded_ls: ursa::cl::BlindedCredentialSecrets,
+    #[serde(alias = "blinded_ms_correctness_proof")]
+    pub blinded_ls_correctness_proof: ursa::cl::BlindedCredentialSecretsCorrectnessProof,
```

Comment:
This adds compatibility for both `blinded_ms` and `blinded_ls` if the spec changes this field.

Reply:
This does however create the output of `blinded_ls` and not `blinded_ms` anymore.

Reply:
Didn't think about the credential request. Not sure if we should change it now, as it's two more fields to update :/ and that we need to support both ways for backwards compatibility

Reply:
With alias we support both for incoming data models, but we always output with `ls`. 

I mainly changed it because blinded_ms that refers to a link secret feels weird. 

Do you think the implications outweigh the naming convention?

Reply:
Oh that's a difficult one, I would really like it to be `ls`. Changing it later would be a breaking change and it feels like an oversight. If we could do it later without a breaking change, I am 100% to just leave it for now.

Reply:
Anoncreds v1 is expected to keep compatibility with what has been done, so it's not likely to change this field. At least not in a breaking manner.